README.rst

```diff
@@ -45,8 +45,8 @@ differences in syntax, naming and conventions:
 - versioneye uses a type, name and version
   https://github.com/versioneye/
 
-- Sonatype Lifecycle uses a format id followed by format specific coordinates. 
-  https://help.sonatype.com/display/NXIQ/Component+Details+API+-+v2  
+- Sonatype Lifecycle uses a format id followed by format specific coordinates.
+  https://help.sonatype.com/display/NXIQ/Component+Details+API+-+v2
 
 
 Solution
```

Comment:
I think for the readme I would lead with the solution rather than the problem so people can see at a glance what this is, they can then dig into the context and problem if they want to understand the why.